main.nf

```diff
@@ -197,22 +190,22 @@ if (params.pizzly) {
              .from(params.readPaths)
              .map { row -> [ row[0], [file(row[1][0])]] }
              .ifEmpty { exit 1, "params.readPaths was empty - no input files supplied" }
-             .into { read_files_fastqc; read_files_star_fusion; read_files_fusioncatcher; read_files_gfusion; 
-                     read_files_fusion_inspector; read_files_ericscript; read_files_pizzly; read_files_squid }
+             .into { read_files_fastqc; read_files_summary; read_files_star_fusion; read_files_fusioncatcher; 
```

Comment:
Just a question, you're getting more an more channels here.
I noticed that you made a break line at some point.
Why not do that for every channel? that way it'll be easier to notice changes.
What do you think?